### DISCUSSION TITLE
Strange error displayed / Specific OGG file triggers "Error during vorbis synthesis -135"

### DISCUSSION THREAD

**Discussion by DevMimas** [godotengine/godot-proposals#8808] [Editor]
Dear Team,
I have copied a project identically and imported it. Then this error message appeared immediately. It could be related to an .ogg file, because such a file is in the project. So it was more of a coincidence that I came across it.
But for me (and I think for some others too) this error message is absolutely meaningless. In my opinion, there should be a much clearer definition of the error and possibly also help on how to fix the problem, if that were possible. Thank you.

![grafik](https://github.com/godotengine/godot-proposals/assets/44207756/0e8cc44a-11e6-4f65-a49b-26da6c2c8f22)

**Comment by theraot** [comment]
If you can provide a small project with just the necessary to reproduce the problem for debug purposes... Please post it as an issue at https://github.com/godotengine/godot/issues

I'm guessing this is related to https://github.com/godotengine/godot/issues/84712 - edit: which was reportedly solved by https://github.com/godotengine/godot/pull/84723

**Reply by DevMimas** [reply]
Thanks. I made a commit in https://github.com/godotengine/godot/pull/84723
